### PR TITLE
Add biocondcutor-alpine and its dependency r-speedglm

### PR DESCRIPTION
**Pull request by upendrak**
* [y ] I have read the guidelines above.
* [y ] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: r-speedglm/bld.bat [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
+@rem Add more build steps here, if they are necessary.
```
Comment (bgruening) [inline r-speedglm/bld.bat]: This file can be removed as we do not build for the Win platform.

File: r-speedglm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "0.3_1"
+
+source:
+  fn: speedglm_0.3-1.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/speedglm_0.3-1.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/speedglm/speedglm_0.3-1.tar.gz
+
```
Comment (bgruening) [inline r-speedglm/meta.yaml]: Can you add a checksum here.

File: r-speedglm/bld.bat (hunk 1 above)
Comment (upendrak) [inline r-speedglm/bld.bat]: Ok got it.

File: r-speedglm/meta.yaml (hunk 2 above)
Comment (upendrak) [inline r-speedglm/meta.yaml]: MD5 (meta.yaml) = 7502ed87a6175f0c94d2fd799370bd39

Comment (bgruening) [inline r-speedglm/meta.yaml]: No from this tarball which you want to download/compile. -> http://cran.r-project.org/src/contrib/speedglm_0.3-1.tar.gz

**Comment by bgruening** [comment]
@upendrak for picard it would be better to create a separate PR. Leave it like it is this time. But separate PR makes it easier to debug if something goes wrong.

**Comment by upendrak** [comment]
Sorry about that last accidental commit. Currently in the process of removing it

**Comment by bgruening** [comment]
Thanks @upendrak!!!

**Comment by upendrak** [comment]
I still don't see the docker images on quay.io. Do i need to wait longer?

**Comment by bgruening** [comment]
It should be there now. It still takes a small amount of time after merging, as travis needs to build the conda package again and the Docker image + pushing to the registry.

**Comment by upendrak** [comment]
Got it. I understand the whole process now. Sorry for the trouble..

**Pull request merged**